lib/mina_sidekiq/tasks.rb

```diff
@@ -179,20 +180,23 @@ def for_each_process(&block)
   task :uninstall do
     case fetch(:init_system)
     when :systemd
-      command %{ systemctl disable #{fetch(:service_unit_name)} }
+      command %{ #{ fetch(:systemctl_command) } disable #{fetch(:service_unit_name)} }
       command %{ rm #{File.join(fetch(:service_unit_path, fetch_systemd_unit_path),fetch(:service_unit_name))}  }
     end
   end
 
   def create_systemd_template
-    template =  "[Unit]\nDescription=sidekiq for #{fetch(:application)} #{fetch(:app_name)}\nAfter=syslog.target network.target\n\n[Service]\nType=simple\nEnvironment=RAILS_ENV=#{ fetch(:rails_env) }\nWorkingDirectory=#{fetch(:deploy_to)}/current\nExecStart=#{fetch(:bundler_path, '/usr/local/bin/bundler')} exec sidekiq -e #{fetch(:rails_env)}\nExecReload=/bin/kill -TSTP $MAINPID\nExecStop=/bin/kill -TERM $MAINPID\n\nRestartSec=1\nRestart=on-failure\n\nSyslogIdentifier=sidekiq\n\n[Install]\nWantedBy=default.target\n"
+    template =  "[Unit]\nDescription=sidekiq for #{fetch(:application)} #{fetch(:app_name)}\nAfter=syslog.target network.target\n\n[Service]\nType=simple\nEnvironment=RAILS_ENV=#{ fetch(:rails_env) }\nStandardOutput=append:#{fetch(:deploy_to)}/current/log/sidekiq.log\nStandardError=append:#{fetch(:deploy_to)}/current/log/sidekiq.log\nWorkingDirectory=#{fetch(:deploy_to)}/current\nExecStart=#{fetch(:bundler_path, '/usr/local/bin/bundler')} exec sidekiq -e #{fetch(:rails_env)}\nExecReload=/bin/kill -TSTP $MAINPID\nExecStop=/bin/kill -TERM $MAINPID\n\nRestartSec=1\nRestart=on-failure\n\nSyslogIdentifier=sidekiq\n\n[Install]\nWantedBy=default.target\n"
```

Comment:
I think it is time to make this a multi line string:

template =  %{
[Unit]
Description=sidekiq for #{fetch(:application)} #{fetch(:app_name)}
# ...
}

Reply:
I've updated PR